### PR TITLE
add caip168 for eip155

### PR DESCRIPTION
**Pull request by zachferland**
https://github.com/ChainAgnostic/CAIPs/pull/168 - links updated after merge of caip

### CODE REVIEW

File: eip155/caip168.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+**Verification Steps**
+
+1) Resolve blockchain transaction payload by `txHash` CID and `chainId`. CID is expected to be DAG-ETH encoded block. 
+2) Get the `data` paramater of the transaction payload, it is expected to be a hex encoded CID. The data value is a byte friendly string and may be prepended with a '0' for a resulting even length hex string. 
+3) Verify that the `root` CID of the timestamp anchor proof is equivalent to the `transaction.data` CID. Equivalence should be checked using a CIDs library, rather than string equivalence, as the CIDs may be encoded in different bases. If they are not equivalent, an error MUST be raised. 
+4) Determine that the transaction has been included in a valid block on the blockchain referenced by the `chainId`. Number of block confirmations for acceptance is up to an implementation. 
+
+
+### `f(32bytes)` Transaction Type Verification
```
Comment (oed) [inline eip155/caip168.md:45]:
```suggestion
### `f(bytes32)` Transaction Type Verification
```

Small typo. This should be `bytes32` like the solidity type.

File: eip155/caip168.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  chainID: "eip155:1"
+  txHash: CID(bagiacgzah24drzou2jlkixpblbgbg6nxfrasoklzttzoht5hixhxz3rlncyq)
+  txType: 'f(32bytes)'
+}
+```
+**Verification Steps**
+
+1) Resolve blockchain transaction payload by `txHash` CID and `chainId`. CID is expected to be DAG-ETH encoded block. 
+2) Get the `data` paramater of the transaction payload, it is expected to be a hex encoded 64 byte string.
+3) The first 32 bytes are the function signature and can be discarded, the next 32 bytes is the first argument of the function, which is expected to be a 32 byte hex encoded partial CID.
```
Comment (oed) [inline eip155/caip168.md:62]: This should be 4 bytes (8 hex chars).

File: eip155/caip168.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+4) The partial CID is a CID with the first two bytes removed, the first byte being the multibase is not needed in byte format, and the second byte being the CID version is not inlcuded, it is assumed to be CID V1. Verify that the `root` CID of the timestamp anchor proof is equivalent to the transaction partial CID. Equivalence should be checked using a CIDs library, and bytes can be compared. If they are not equivalent, an error MUST be raised.
```
Comment (oed) [inline eip155/caip168.md:63]: We actually include the entire multihash, but not the CID version nor the ipld codec (which is dag-cbor).

File: eip155/caip168.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+3) Verify that the `root` CID of the timestamp anchor proof is equivalent to the `transaction.data` CID. Equivalence should be checked using a CIDs library, rather than string equivalence, as the CIDs may be encoded in different bases. If they are not equivalent, an error MUST be raised. 
+4) Determine that the transaction has been included in a valid block on the blockchain referenced by the `chainId`. Number of block confirmations for acceptance is up to an implementation. 
+
+
+### `f(32bytes)` Transaction Type Verification
+
+This section describes transaction verification when a timestamp anchor proof references a `chainId` in the `eip155` namespace and `txType` is `f(32bytes)`. Type `f(32bytes)` refers to a transaction which makes a function call on a contract with the function signature including a single 32 bytes argument. The 32 byte argument is the merkle root CID. A CID is more than 32bytes, and a partial CID is used to allow the argument to be efficiently packed in the transaction. Function name or contract does not matter, and is up to an implementation.
```
Comment (oed) [inline eip155/caip168.md:47]: Please mention that `f(bytes32)` assumes that a `dag-cbor` CID is being used.

File: eip155/caip168.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
+---
+namespace-identifier: eip155-caip168
+title: EVM Namespace - IPLD Timestamp Proof
+author: Zach Ferland (@zachferland)
+discussions-to: <-->
```
Comment (oed) [inline eip155/caip168.md:5]:
```suggestion
discussions-to: https://github.com/ChainAgnostic/namespaces/pull/42
```

**Review by oed — CHANGES_REQUESTED** [review]
A few minor changes needed.

**Comment by zachferland** [comment]
editorial pass merged

**Pull request merged**